pkg/util/timeconv/duration/duration.go

```diff
@@ -0,0 +1,161 @@
+package duration
```

Comment:
since we already have a `ParseDuration` func with the same prototype in the parent `timeconv` package, which this one perfectly subsumes and extends; we should just replace that original function rather than introducing another package altogether that does the same thing.